docs/index.md

```diff
@@ -1377,6 +1378,27 @@ assert.near.message = "Expected ${actual} to be equal to ${expected} +/- ${delta
 refute.near.message = "Expected ${actual} not to be equal to ${expected} +/- ${delta}";
 ```
 
+### `hasArity()`
+
+```js
+assert.hasArity(actual, prototype[, message])
```

Comment:
```suggestion
assert.hasArity(actual, arity[, message])
```

Reply:
Well spotted copy pasta! I've pushed a commit that fixes this